

<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>AlphaFold &mdash; NBD Docs 1.0.0 documentation</title>
  

  
  <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />

  
  
  
  

  
  <!--[if lt IE 9]>
    <script src="../../_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
        <script src="../../_static/jquery.js"></script>
        <script src="../../_static/underscore.js"></script>
        <script src="../../_static/doctools.js"></script>
        <script src="../../_static/language_data.js"></script>
    
    <script type="text/javascript" src="../../_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Protein-protein docking" href="../ppi/ppi.html" />
    <link rel="prev" title="New to NBD? Set everything up!" href="../new_comers/index.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="../../index.html" class="icon icon-home" alt="Documentation Home"> NBD Docs
          

          
          </a>

          
            
            
              <div class="version">
                1.0
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <ul>
<li class="toctree-l1"><a class="reference internal" href="../../databases/index.html">Compound DataBases</a></li>
</ul>
<ul class="current">
<li class="toctree-l1 current"><a class="reference internal" href="../index.html">Software</a><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="../index.html#summary">Summary</a></li>
<li class="toctree-l2"><a class="reference internal" href="../index.html#job-launching-templates">Job launching Templates</a></li>
<li class="toctree-l2"><a class="reference internal" href="../index.html#pele">PELE</a></li>
<li class="toctree-l2"><a class="reference internal" href="../index.html#namd">NAMD</a></li>
<li class="toctree-l2"><a class="reference internal" href="../index.html#schrodinger">Schrodinger</a></li>
<li class="toctree-l2"><a class="reference internal" href="../index.html#python-packages">Python packages</a></li>
<li class="toctree-l2"><a class="reference internal" href="../index.html#gromacs">GROMACS</a></li>
<li class="toctree-l2"><a class="reference internal" href="../index.html#plumed">PLUMED</a></li>
<li class="toctree-l2"><a class="reference internal" href="../index.html#nice-dcv">Nice DCV</a></li>
<li class="toctree-l2"><a class="reference internal" href="../index.html#tests">Tests</a></li>
<li class="toctree-l2"><a class="reference internal" href="../index.html#newcomers">NewComers</a></li>
<li class="toctree-l2 current"><a class="reference internal" href="../index.html#alphafold">AlphaFold</a><ul class="current">
<li class="toctree-l3 current"><a class="current reference internal" href="#">AlphaFold</a></li>
<li class="toctree-l3"><a class="reference internal" href="#running-alphafold-within-singularity">Running AlphaFold within Singularity</a></li>
<li class="toctree-l3"><a class="reference internal" href="#alphafold-output">AlphaFold output</a></li>
<li class="toctree-l3"><a class="reference internal" href="#running-alphafold-multimer">Running AlphaFold Multimer</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#example">Example</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../index.html#protein-protein-docking">Protein-protein docking</a></li>
</ul>
</li>
</ul>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../compute_power/index.html">IT infrastructure</a></li>
</ul>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../papers/index.html">Articles</a></li>
</ul>

            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">NBD Docs</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../../index.html" class="icon icon-home"></a> &raquo;</li>
        
          <li><a href="../index.html">Software</a> &raquo;</li>
        
      <li>AlphaFold</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="../../_sources/software/alphafold/alphafold.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="alphafold">
<h1>AlphaFold<a class="headerlink" href="#alphafold" title="Permalink to this headline">¶</a></h1>
<p>This documentation provides the information and templates to run <a class="reference external" href="https://github.com/deepmind/alphafold">Alpha Fold</a>.</p>
<p>The parameters needed to run Alpha Fold are:</p>
<blockquote>
<div><ul class="simple">
<li><p><strong>ALPHAFOLD_DATA_PATH:</strong> Absolute path to folder with databases.</p></li>
<li><p><strong>ALPHAFOLD_MODELS:</strong> Absolute path to folder with models.</p></li>
<li><p><strong>pwd:</strong> Path to Singularity Image File (SIF) file.</p></li>
<li><p><strong>fasta_paths:</strong> Path to the input sequence in fasta format.</p></li>
<li><p><strong>uniref90_database_path:</strong> Path to Uniref90 database for use by JackHMMER.</p></li>
<li><p><strong>mgnify_database_path:</strong> Path to the MGnify database for use by JackHMMER.</p></li>
<li><p><strong>bfd_database_path:</strong> Path to the BFD database for use by HHblits.</p></li>
<li><p><strong>uniclust30_database_path:</strong> Path to Uniclust30 database for use by HHblits.</p></li>
<li><p><strong>pdb70_database_path:</strong> Path to PDB70 database for use by HHsearch.</p></li>
<li><p><strong>template_mmcif_dir_</strong> Path to a directory with template mmCIF structures, each named &lt;pdb_id&gt;.cif.</p></li>
<li><p><strong>uniprot_database_path</strong> Path to the UniProt database for AlphaFold Multimer.</p></li>
<li><p><strong>obsolete_pdbs_path:</strong> Path to a file mapping obsolete PDB IDs to their replacements.</p></li>
<li><p><strong>max_template_date:</strong> Maximum template release date to consider (ISO-8601 format - i.e. YYYY-MM-DD). Important if folding historical test sets. Default is None.</p></li>
<li><p><strong>output_dir:</strong> Path to a directory that will store the results.</p></li>
<li><p><strong>model_preset:</strong> [‘monomer’,’monomer_casp14’,’monomer_ptm’,’multimer’]. Control which AlphaFold model use, choosing between the original model used at CASP14 with no ensembling (monomer). the original model used at CASP14 with num_ensemble=8, matching our CASP14 configuration. This is largely provided for reproducibility as it is 8x more computationally expensive for limited accuracy gain (+0.1 average GDT gain on CASP14 domains) (monomer_casp14). the original CASP14 model fine tuned with the pTM head, providing a pairwise confidence measure. It is slightly less accurate than the normal monomer model (‘monomer_ptm’). The AlphaFold-Multimer model, to use this model, provide a multi-sequence FASTA file. In addition, the UniProt database should have been downloaded (‘multimer’).</p></li>
<li><p><strong>db_preset:</strong> [‘reduced_dbs’, ‘full_dbs’, ‘casp14’]. Choose preset model configuration - no ensembling and smaller genetic database config (reduced_dbs), no ensembling and full genetic database config (full_dbs) or full genetic database config and 8 model ensemblings (casp14). Default is full_dbs.</p></li>
<li><p><strong>benchmark:</strong> [True, False]. Run multiple JAX model evaluations to obtain a timing that excludes the compilation time, which should be more indicative of the time required for inferencing many proteins. Default is False.</p></li>
</ul>
</div></blockquote>
<dl class="field-list simple">
<dt class="field-odd">Important</dt>
<dd class="field-odd"><p>The reference databases and models were downloaded in the directory <strong>/shared/work/NBD_Utilities/AlphaFold/databases</strong> and the singularity image file (alphafold_2.1.0.sif) of AlphaFold is available at <strong>/shared/work/NBD_Utilities/AlphaFold</strong></p>
</dd>
<dt class="field-even">Important</dt>
<dd class="field-even"><p>The max_template_date flag is mandatory when running AlphaFold. If you are predicting the structure of a protein that is already in PDB and you wish to avoid using it as a template, then max_template_date must be set to be before the release date of the structure. If you do not need to specify a date, by default we set today’s date. For example, if we are running the simulation on August 7th 2021, we set – max_template_date = 2021-08-07.</p>
</dd>
</dl>
</div>
<div class="section" id="running-alphafold-within-singularity">
<h1>Running AlphaFold within Singularity<a class="headerlink" href="#running-alphafold-within-singularity" title="Permalink to this headline">¶</a></h1>
<p>Here is an example on how to run AlphaFold.
First, we need a protein sequence in FASTA format.</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="o">&gt;</span><span class="mi">5</span><span class="n">ZE6_1</span>
<span class="n">MNLEKINELTAQDMAGVNAAILEQLNSDVQLINQLGYYIVSGGGKRIRPMIAVLAARAVGYEGNAHVTIAALIEFIHTATLLHDDVVDESDMRRGKATANAA</span>
<span class="n">FGNAASVLVGDFIYTRAFQMMTSLGSLKVLEVMSEAVNVIAEGEVLQLMNVNDPDITEENYMRVIYSKTARLFEAAAQCSGILAGCTPEEEKGLQDYGRYLG</span>
<span class="n">TAFQLIDDLLDYNADGEQLGKNVGDDLNEGKPTLPLLHAMHHGTPEQAQMIRTAIEQGNGRHLLEPVLEAMNACGSLEWTRQRAEEEADKAIAALQVLPDTP</span>
<span class="n">WREALIGLAHIAVQRDR</span>
</pre></div>
</div>
<p>If we want to run AlphaFold in, for example, the directory <code class="docutils literal notranslate"><span class="pre">AlphaFold/test</span></code></p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>user@login01:~$  cd AlphaFold/test
user@login01:AlphaFold/test$  mkdir alphafold_output # Create the directory for AlphaFold output
user@login01:AlphaFold/test$  ls # The directory should contain the singularity image file (.sif) and the input FASTA sequence
alphafold_output alphafold_2.1.0.sif input.fasta
</pre></div>
</div>
<dl class="simple">
<dt>To run Alpha Fold, please change in the following template:</dt><dd><ul class="simple">
<li><p>output_dir</p></li>
<li><p>fasta_paths</p></li>
</ul>
</dd>
</dl>
<p><strong>Alpha Fold - v2.0.1 - Template to run</strong></p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">The</span> <span class="n">memory</span> <span class="n">needed</span> <span class="k">for</span> <span class="n">a</span> <span class="n">job</span> <span class="n">depends</span> <span class="n">on</span> <span class="n">the</span> <span class="n">length</span> <span class="n">of</span> <span class="n">the</span> <span class="nb">input</span> <span class="n">FASTA</span> <span class="n">sequence</span> <span class="ow">and</span> <span class="n">the</span> <span class="n">number</span> <span class="n">of</span> <span class="n">models</span> <span class="n">used</span><span class="o">.</span>
<span class="n">Consider</span> <span class="n">increasing</span> <span class="n">the</span> <span class="n">memory</span> <span class="k">if</span> <span class="n">you</span> <span class="n">are</span> <span class="n">working</span> <span class="k">with</span> <span class="n">a</span> <span class="n">large</span> <span class="n">sequence</span> <span class="ow">or</span> <span class="k">with</span> <span class="nb">all</span> <span class="n">the</span> <span class="n">models</span><span class="o">.</span>
</pre></div>
</div>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="ch">#!/bin/bash</span>
<span class="c1">#SBATCH --job-name alphafold-run</span>
<span class="c1">#SBATCH --gres=gpu:1</span>
<span class="c1">#SBATCH --cpus-per-task=8</span>
<span class="c1">#SBATCH --mem=40G</span>

<span class="c1">#set the environment PATH</span>
<span class="nb">export</span> <span class="nv">PYTHONNOUSERSITE</span><span class="o">=</span>True
<span class="nv">ALPHAFOLD_DATA_PATH</span><span class="o">=</span>/shared/work/NBD_Utilities/AlphaFold/databases
<span class="nv">ALPHAFOLD_MODELS</span><span class="o">=</span>/shared/work/NBD_Utilities/AlphaFold/databases/params

module load CUDA/9.2.88-iccifort-2018.1.163-GCC-6.4.0-2.28
module load cuDNN
<span class="nb">export</span> <span class="nv">CUDA_VISIBLE_DEVICES</span><span class="o">=</span>-1

<span class="c1">#Run the command</span>
singularity run --nv <span class="se">\</span>
 -B <span class="nv">$ALPHAFOLD_DATA_PATH</span>:/data <span class="se">\</span>
 -B <span class="nv">$ALPHAFOLD_MODELS</span> <span class="se">\</span>
 -B .:/etc <span class="se">\</span>
 --pwd  /app/alphafold alphafold_2.1.0.sif <span class="se">\</span>
 --fasta_paths<span class="o">=</span>/path/to/input/sequence/input.fasta  <span class="se">\</span>
 --uniref90_database_path<span class="o">=</span>/data/uniref90/uniref90.fasta  <span class="se">\</span>
 --data_dir<span class="o">=</span>/data <span class="se">\</span>
 --mgnify_database_path<span class="o">=</span>/data/mgnify/mgy_clusters.fa   <span class="se">\</span>
 --bfd_database_path<span class="o">=</span>/data/bfd/bfd_metaclust_clu_complete_id30_c90_final_seq.sorted_opt <span class="se">\</span>
 --uniclust30_database_path<span class="o">=</span>/data/uniclust30/uniclust30_2018_08/uniclust30_2018_08 <span class="se">\</span>
 --pdb70_database_path<span class="o">=</span>/data/pdb70/pdb70  <span class="se">\</span>
 --template_mmcif_dir<span class="o">=</span>/data/pdb_mmcif/mmcif_files  <span class="se">\</span>
 --obsolete_pdbs_path<span class="o">=</span>/data/pdb_mmcif/obsolete.dat <span class="se">\</span>
 --max_template_date<span class="o">=</span> YYYY-MM-DD <span class="se">\</span>
 --output_dir<span class="o">=</span>/path/to/output/directory  <span class="se">\</span>
 --model_preset<span class="o">=</span><span class="s1">&#39;monomer&#39;</span>
</pre></div>
</div>
</div>
<div class="section" id="alphafold-output">
<h1>AlphaFold output<a class="headerlink" href="#alphafold-output" title="Permalink to this headline">¶</a></h1>
<p>The outputs will be in a subfolder of <cite>output_dir</cite>. They
include the computed MSAs, unrelaxed structures, relaxed structures, ranked
structures, raw model outputs, prediction metadata, and section timings. The
<cite>output_dir</cite> directory will have the following structure:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="o">&lt;</span><span class="n">target_name</span><span class="o">&gt;/</span>
    <span class="o">|-</span> <span class="nb">input</span><span class="o">/</span>
       <span class="o">|-</span> <span class="n">features</span><span class="o">.</span><span class="n">pkl</span>
       <span class="o">|-</span> <span class="n">ranked_</span><span class="p">{</span><span class="mi">0</span><span class="p">,</span><span class="mi">1</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span><span class="mi">4</span><span class="p">}</span><span class="o">.</span><span class="n">pdb</span>
       <span class="o">|-</span> <span class="n">ranking_debug</span><span class="o">.</span><span class="n">json</span>
       <span class="o">|-</span> <span class="n">relaxed_model_</span><span class="p">{</span><span class="mi">1</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span><span class="mi">4</span><span class="p">,</span><span class="mi">5</span><span class="p">}</span><span class="o">.</span><span class="n">pdb</span>
       <span class="o">|-</span> <span class="n">result_model_</span><span class="p">{</span><span class="mi">1</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span><span class="mi">4</span><span class="p">,</span><span class="mi">5</span><span class="p">}</span><span class="o">.</span><span class="n">pkl</span>
       <span class="o">|-</span> <span class="n">timings</span><span class="o">.</span><span class="n">json</span>
       <span class="o">|-</span> <span class="n">unrelaxed_model_</span><span class="p">{</span><span class="mi">1</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span><span class="mi">3</span><span class="p">,</span><span class="mi">4</span><span class="p">,</span><span class="mi">5</span><span class="p">}</span><span class="o">.</span><span class="n">pdb</span>
       <span class="o">|-</span> <span class="n">msas</span><span class="o">/</span>
          <span class="o">|-</span> <span class="n">bfd_uniclust_hits</span><span class="o">.</span><span class="n">a3m</span>
          <span class="o">|-</span> <span class="n">mgnify_hits</span><span class="o">.</span><span class="n">sto</span>
          <span class="o">|-</span> <span class="n">uniref90_hits</span><span class="o">.</span><span class="n">sto</span>
</pre></div>
</div>
<p>The contents of each output file are as follows:</p>
<ul class="simple">
<li><p><strong>features.pkl:</strong> A pickle file containing the input feature NumPy arrays
used by the models to produce the structures.</p></li>
<li><p><strong>unrelaxed_model_x.pdb:</strong> A PDB format text file containing the predicted
structure, exactly as outputted by the model.</p></li>
<li><p><strong>relaxed_model_x.pdb:</strong> A PDB format text file containing the predicted
structure, after performing an Amber relaxation procedure on the unrelaxed
structure prediction (see Jumper et al. 2021, Suppl. Methods 1.8.6 for
details).</p></li>
<li><p><strong>ranked_x.pdb:</strong> A PDB format text file containing the relaxed predicted
structures, after reordering by model confidence. Here <cite>ranked_0.pdb</cite> should
contain the prediction with the highest confidence, and <cite>ranked_4.pdb</cite> the
prediction with the lowest confidence. To rank model confidence, we use
predicted LDDT (pLDDT) scores (see Jumper et al. 2021, Suppl. Methods 1.9.6
for details).</p></li>
<li><p><strong>ranking_debug.json:</strong> A JSON format text file containing the pLDDT values
used to perform the model ranking, and a mapping back to the original model
names.</p></li>
<li><p><strong>timings.json:</strong> A JSON format text file containing the times taken to run
each section of the AlphaFold pipeline.</p></li>
<li><p><strong>msas/:</strong> - A directory containing the files describing the various genetic
tool hits that were used to construct the input MSA.</p></li>
<li><p><strong>result_model_x.pkl:</strong> A <cite>pickle</cite> file containing a nested dictionary of the
various NumPy arrays directly produced by the model. In addition to the
output of the structure module, this includes auxiliary outputs such as:</p>
<ul>
<li><p>Distograms (<strong>distogram/logits</strong> contains a NumPy array of shape [N_res,
N_res, N_bins] and <strong>distogram/bin_edges</strong> contains the definition of the
bins).</p></li>
<li><p>Per-residue pLDDT scores (<strong>plddt</strong> contains a NumPy array of shape
[N_res] with the range of possible values from 0 to 100, where 100
means most confident). This can serve to identify sequence regions
predicted with high confidence or as an overall per-target confidence
score when averaged across residues.</p></li>
<li><p>Present only if using pTM models: predicted TM-score (<strong>ptm</strong> field
contains a scalar). As a predictor of a global superposition metric,
this score is designed to also assess whether the model is confident in
the overall domain packing.</p></li>
<li><p>Present only if using pTM models: predicted pairwise aligned errors
(<strong>predicted_aligned_error</strong> contains a NumPy array of shape [N_res,
N_res] with the range of possible values from 0 to
<strong>max_predicted_aligned_error</strong>, where 0 means most confident). This can
serve for a visualisation of domain packing confidence within the
structure.</p></li>
</ul>
</li>
</ul>
</div>
<div class="section" id="running-alphafold-multimer">
<h1>Running AlphaFold Multimer<a class="headerlink" href="#running-alphafold-multimer" title="Permalink to this headline">¶</a></h1>
<p>The steps are the same as when folding a monomer, but it is needed to provide:</p>
<ol class="arabic simple">
<li><p>An input <a href="#id1"><span class="problematic" id="id2">`</span></a>fasta file with multiple sequences.</p></li>
<li><p>Set the <strong>–model-preset</strong> flag to ‘multimer’.</p></li>
<li><p>Optionally set the <strong>–is_prokaryote_list</strong> flag with booleans that determine whether all input sequences in the given <cite>fasta</cite> file are prokaryotic. If that is not the case or the origin is unknown, set to <cite>false</cite> for that <cite>fasta</cite>.</p></li>
</ol>
<div class="section" id="example">
<h2>Example<a class="headerlink" href="#example" title="Permalink to this headline">¶</a></h2>
<p>In this tutorial we will fold a multimer using AlphaFold. We will be using a Human GITR-GITRL complex (PDB ID: 7KHD).</p>
<p>1. Sequence file preparation:
The multimer sequence can be download from the PDB databse.</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="o">&gt;</span><span class="mi">7</span><span class="n">KHD_1</span><span class="o">|</span><span class="n">Chains</span> <span class="n">A</span><span class="p">,</span> <span class="n">B</span><span class="o">|</span><span class="n">Tumor</span> <span class="n">necrosis</span> <span class="n">factor</span> <span class="n">ligand</span> <span class="n">superfamily</span> <span class="n">member</span> <span class="mi">18</span><span class="o">|</span><span class="n">Homo</span> <span class="n">sapiens</span> <span class="p">(</span><span class="mi">9606</span><span class="p">)</span>
<span class="n">QLETAKEPCMAKFGPLPSKWQMASSEPPCVNKVSDWKLEILQNGLYLIYGQVAPNANYNDVAPFEVRLYKNKDMIQTLTNKSKIQNVGGTYELHVGDTIDLIFNSEHQVLKNNTYWGIILLANPQFIS</span>
<span class="o">&gt;</span><span class="mi">7</span><span class="n">KHD_2</span><span class="o">|</span><span class="n">Chains</span> <span class="n">C</span><span class="p">,</span> <span class="n">D</span><span class="o">|</span><span class="n">Tumor</span> <span class="n">necrosis</span> <span class="n">factor</span> <span class="n">receptor</span> <span class="n">superfamily</span> <span class="n">member</span> <span class="mi">18</span><span class="o">|</span><span class="n">Homo</span> <span class="n">sapiens</span> <span class="p">(</span><span class="mi">9606</span><span class="p">)</span>
<span class="n">QRPTGGPGCGPGRLLLGTGTDARCCRVHTTRCCRDYPGEECCSEWDCMCVQPEFHCGDPCCTTCRHHPCPPGQGVQSQGKFSFGFQCIDCASGTFSGGHEGHCKPWTDCTQFGFLTVFPGNKTHNAVCVPGSPPAEP</span>
</pre></div>
</div>
<p>If the multimer has repeated chains,the input fasta should be</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="o">&gt;</span><span class="mi">7</span><span class="n">KHD_1</span><span class="o">|</span><span class="n">Chain</span> <span class="n">A</span>
<span class="n">QLETAKEPCMAKFGPLPSKWQMASSEPPCVNKVSDWKLEILQNGLYLIYGQVAPNANYNDVAPFEVRLYKNKDMIQTLTNKSKIQNVGGTYELHVGDTIDLIFNSEHQVLKNNTYWGIILLANPQFIS</span>
<span class="o">&gt;</span><span class="mi">7</span><span class="n">KHD_2</span><span class="o">|</span><span class="n">Chain</span> <span class="n">B</span>
<span class="n">QLETAKEPCMAKFGPLPSKWQMASSEPPCVNKVSDWKLEILQNGLYLIYGQVAPNANYNDVAPFEVRLYKNKDMIQTLTNKSKIQNVGGTYELHVGDTIDLIFNSEHQVLKNNTYWGIILLANPQFIS</span>
<span class="o">&gt;</span><span class="mi">7</span><span class="n">KHD_3</span><span class="o">|</span><span class="n">Chain</span> <span class="n">C</span>
<span class="n">QRPTGGPGCGPGRLLLGTGTDARCCRVHTTRCCRDYPGEECCSEWDCMCVQPEFHCGDPCCTTCRHHPCPPGQGVQSQGKFSFGFQCIDCASGTFSGGHEGHCKPWTDCTQFGFLTVFPGNKTHNAVCVPGSPPAEP</span>
<span class="o">&gt;</span><span class="mi">7</span><span class="n">KHD_4</span><span class="o">|</span><span class="n">Chain</span> <span class="n">D</span>
<span class="n">QRPTGGPGCGPGRLLLGTGTDARCCRVHTTRCCRDYPGEECCSEWDCMCVQPEFHCGDPCCTTCRHHPCPPGQGVQSQGKFSFGFQCIDCASGTFSGGHEGHCKPWTDCTQFGFLTVFPGNKTHNAVCVPGSPPAEP</span>
</pre></div>
</div>
<p>In our multimer, chains A-B and chains C-D are repeated.</p>
<p>Then, submit the following sh file:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">When</span> <span class="n">running</span> <span class="n">AlphaFold</span> <span class="n">Multimer</span><span class="p">,</span> <span class="n">it</span> <span class="ow">is</span> <span class="n">needed</span> <span class="n">to</span> <span class="n">define</span> <span class="n">UniProt</span><span class="s1">&#39;s database file.</span>
</pre></div>
</div>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="ch">#!/bin/bash</span>
<span class="c1">#SBATCH --job-name af_multimer</span>
<span class="c1">#SBATCH --cpus-per-task=8</span>
<span class="c1">#SBATCH --mem=20G</span>

<span class="c1">#set the environment PATH</span>
<span class="nb">export</span> <span class="nv">PYTHONNOUSERSITE</span><span class="o">=</span>True
<span class="nv">ALPHAFOLD_DATA_PATH</span><span class="o">=</span>/shared/work/NBD_Utilities/AlphaFold/databases
<span class="nv">ALPHAFOLD_MODELS</span><span class="o">=</span>/shared/work/NBD_Utilities/AlphaFold/databases/params

module purge
module load CUDA/9.2.88-iccifort-2018.1.163-GCC-6.4.0-2.28
module load cuDNN
<span class="nb">export</span> <span class="nv">CUDA_VISIBLE_DEVICES</span><span class="o">=</span>-1

<span class="c1">#Run the command</span>
singularity run --nv <span class="se">\</span>
 -B <span class="nv">$ALPHAFOLD_DATA_PATH</span>:/data <span class="se">\</span>
 -B <span class="nv">$ALPHAFOLD_MODELS</span> <span class="se">\</span>
 -B .:/etc <span class="se">\</span>
 --pwd  /app/alphafold  alphafold_2.1.0.sif <span class="se">\</span>
 --data_dir<span class="o">=</span>/data <span class="se">\</span>
 --fasta_paths<span class="o">=</span>/shared/work/NBD_Utilities/AlphaFold/test_container/af_multimer/sep_chains/7khd.fasta <span class="se">\</span>
 --uniref90_database_path<span class="o">=</span>/data/uniref90/uniref90.fasta  <span class="se">\</span>
 --data_dir<span class="o">=</span>/data <span class="se">\</span>
 --mgnify_database_path<span class="o">=</span>/data/mgnify/mgy_clusters.fa   <span class="se">\</span>
 --bfd_database_path<span class="o">=</span>/data/bfd/bfd_metaclust_clu_complete_id30_c90_final_seq.sorted_opt <span class="se">\</span>
 --uniclust30_database_path<span class="o">=</span>/data/uniclust30/uniclust30_2018_08/uniclust30_2018_08 <span class="se">\</span>
 --pdb_seqres_database_path<span class="o">=</span>/data/pdb_seqres/pdb_seqres.txt <span class="se">\</span>
 --uniprot_database_path<span class="o">=</span>/data/uniprot/uniprot.fasta <span class="se">\</span>
 --template_mmcif_dir<span class="o">=</span>/data/pdb_mmcif/mmcif_files  <span class="se">\</span>
 --obsolete_pdbs_path<span class="o">=</span>/data/pdb_mmcif/obsolete.dat <span class="se">\</span>
 --max_template_date<span class="o">=</span><span class="m">2021</span>-03-03 <span class="se">\</span>
 --model_preset<span class="o">=</span><span class="s1">&#39;multimer&#39;</span> <span class="se">\</span>
 --output_dir<span class="o">=</span>/shared/work/NBD_Utilities/AlphaFold/test_container/af_multimer/sep_chains/7khd
</pre></div>
</div>
</div>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="../ppi/ppi.html" class="btn btn-neutral float-right" title="Protein-protein docking" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="../new_comers/index.html" class="btn btn-neutral float-left" title="New to NBD? Set everything up!" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        
        &copy; Copyright 2021, NBD | Nostrum Biodiscovery

    </p>
  </div>
    
    
    
    Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>